src/screens/Dashboard/TripSearch.tsx

```diff
@@ -556,7 +563,8 @@ function computeNoResultReasons(
 export const useShouldShowTravelSearchFilterOnboarding = () => {
   const [shouldShow, setShouldShow] = useState(false);
   const isFocused = useIsFocused();
-  const enabled = useTravelSearchFiltersEnabled();
+  const travelSearchFiltersState = useTravelSearchFiltersState();
+  const enabled = travelSearchFiltersState.enabled;
```

Comment:
Is this cleaner?

```suggestion
  const {enabled} = useTravelSearchFiltersState();
```


or

```
  const {enabled: filtersEnabled} = useTravelSearchFiltersState();
```